account_bank_statement_import/account_bank_statement_import.py

```diff
@@ -69,15 +72,35 @@ def import_file(self):
 
     @api.model
     def _import_file(self, data_file):
-        """ Create bank statement(s) from file
-        """
+        """ Create bank statement(s) from file."""
         # The appropriate implementation module returns the required data
-        currency_code, account_number, stmts_vals = self._parse_file(data_file)
-        # Check raw data
-        self._check_parsed_data(stmts_vals)
+        statement_ids = []
+        notifications = []
+        statements = self._parse_file(data_file)
```

Comment:
Hello Holger, this is a concious change. Parsing can return data for multiple statements, possibly for different currencies and bank accounts.